lib/tasks/github.rake

```diff
@@ -14,34 +14,13 @@ namespace :github do
     # This list was directly taken from the Popular Languages select box on
     # GitHub's advanced search page: https://github.com/search/advanced
     POPULAR_LANGUAGES = [
-      'ActionScript',
-      'C',
       'C#',
-      'C++',
-      'Clojure',
-      'CoffeeScript',
-      'CSS',
-      'Go',
-      'Haskell',
-      'HTML',
-      'Java',
       'JavaScript',
-      'Lua',
-      'MATLAB',
-      'Objective-C',
-      'Perl',
-      'PHP',
-      'Python',
-      'R',
-      'Ruby',
-      'Scala',
-      'Shell',
-      'Swift',
-      'TeX',
-      'Vim script'
+      'Python'
```

Comment:
Was looking at https://octoverse.github.com/ when making a judgement call about which languages to include. Also, things like `PHP` violate some of the database constraints we have in place in `schema.rb` and will stop the rake altogether.